### PR TITLE
JW-98: rt port

### PR DESCRIPTION
**Pull request by sodar**
* [x] WIP: Before review https://github.com/codilime/contrail-vrouter/pull/82 must be merged.

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
Retest this pleaser

**Comment by sodar** [comment]
Retest this please

File: utils/rt.c [hunk 1]
```diff
@@ -24,6 +25,10 @@
 #else
 #include <net/ethernet.h>
 #endif
+#else
+#include <wingetopt.h>
+#include "vr_os.h"
```
Comment (michal-clapinski) [inline utils/rt.c]: This doesn't have to be inside #ifdef

File: .gitignore [hunk 2]
```diff
@@ -28,6 +28,7 @@ modules.order
 .vs
 *.VC.db
 *.VC.VC.opendb
+/utils/windows/*/x64
```
Comment (michal-clapinski) [inline .gitignore]: What is this for exactly?

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

File: utils/vr_win_util.c [hunk 3]
```diff
@@ -143,8 +145,26 @@ struct ether_addr *
...
     return ether_aton_r(asc, &addr);
 }
+
+char *
+ether_ntoa(const struct ether_addr *addr)
+{
+    static char buffer[ETHER_ADDR_STR_LEN];
+
+    memset(buffer, 0, sizeof(buffer));
+    int ret = snprintf(buffer, sizeof(buffer), "%02x:%02x:%02x:%02x:%02x:%02x",
```
Comment (Attil) [inline utils/vr_win_util.c]: Please look at `vr_defs.h`. You have things like `MAC_FORMAT` and `MAC_VALUE` there, which could be used here.

**Comment by michal-clapinski** [comment]
Those MAC* macros are pretty cool. Nice catch @attil

**Pull request merged**